crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -266,19 +267,35 @@ def repos_path
       Pathname.new(File.join("/srv/tftpboot", @platform, @arch, "repos"))
     end
 
+    def repo_path
+      repos_path.join(name)
+    end
+
+    def smt_path
+      unless @config["smt_path"].blank?
```

Comment:
I'll do the `Pathname.new().join()` bit, but I'm not a big fan of the multiline style. Is this somehow recommended by some guidelines?